sys/suit/handlers_command_seq.c

```diff
@@ -316,13 +322,16 @@ static int _dtv_fetch(suit_manifest_t *manifest, int key,
         return res;
     }
 
+    LOG_INFO("Updating flag\n");
+    suit_component_set_flag(comp, SUIT_COMPONENT_FLAG_FETCHED);
+
     if (res) {
+        suit_component_set_flag(comp, SUIT_COMPONENT_FLAG_FETCH_FAILED);
+        /* TODO: purge component? */
```

Comment:
Is this needed?

Reply:
It's a note to remind me to implement a way to remove a component if either the fetch or the digest verification failed. It's not different than what we do now.

Reply:
But is purging the component needed? AFAIU whatever is not in the running slot is basically garbage

Reply:
It could also be malicious code inserted by an attacker. This is of course still a first stage in the attack, it requires another vulnerability to jump to the inserted code

Reply:
better be explicit? shouldn't leave the slot in an attacker controlled state. (but folo)

Reply:
Clarified the comment a bit